### PR TITLE
feat: add css only logo carousel

### PR DESCRIPTION
**Pull request by cmunns**
https://user-images.githubusercontent.com/1189523/195168999-e3f2b96d-18fd-420a-9cd6-fb2978b0ebc5.mov

### CODE REVIEW

File: packages/website/pages/pricing.js [hunk 1]
```diff
@@ -53,18 +53,147 @@ const PricingHeader = () => (
...
-            <div className="image-label"></div>
-          </div>
-          <div className="block image-block">
-            <Img src="/images/logos/filecoin.svg" alt="Filecoin" />
-            <div className="image-label"></div>
+        <div className="wrapping-logos carousel">
+          <div className="wrapping-logos-inner">
+            <div className="wrapping-logos-images">
+              <div className="block image-block">
+                <Img src="/images/logos/nft-storage.png" alt="NFT Storage" layout="fill" />
```
Comment (drewdelano) [inline packages/website/pages/pricing.js:60]: Might be nice if this was driven from an array of logos, but I think this is fine too.

File: package-lock.json [hunk 2]
```diff
@@ -36331,7 +36333,7 @@
         "@web3-storage/db": "^4.0.0",
         "@web3-storage/multipart-parser": "^1.0.0",
         "@web3-storage/tools": "^1.0.0",
-        "@web3-storage/toucan-js": "*",
+        "@web3-storage/toucan-js": "^2.7.1",
```
Comment (drewdelano) [inline package-lock.json:36336]: I can't tell if this is concerning or not.

**Comment by heyjay44** [comment]
@cmunns is it possible to have it look less jittery. Maybe it was my screen but the logos were hard to read.

**Comment by cmunns** [comment]
> @cmunns is it possible to have it look less jittery. Maybe it was my screen but the logos were hard to read.

Can you upload a video of what you're seeing by chance?

File: package-lock.json (hunk 2 above)
Comment (cmunns) [reply package-lock.json:36336]: Probably just from pulling latest and installing deps?

File: packages/website/pages/pricing.js (hunk 1 above)
Comment (cmunns) [reply packages/website/pages/pricing.js:60]: great idea actually! bc the set repeats itself so this would be handy.

**Comment by gobengo** [comment]
To resolve the conflicts with main, all I had to do was `npm install package-lock-only` https://stackoverflow.com/questions/55599356/what-does-npm-i-package-lock-only-do

**Comment by gobengo** [comment]
Lookin good on https://staging.web3.storage/pricing/

**Pull request merged**